autobot/telegram/parser.py

```diff
@@ -814,14 +814,7 @@ def _parse_poll(self,key:str,val:dict) -> Poll|PollAnswer|PollOption:
         if (key == "poll"):
             poll_obj = Poll()
             for k,v in val.items():
-                match k:
-                    case "options":
-                        options = []
-                        for option in v:
-                            options.append(self._parse_polloption(k,option))
-                        setattr(poll_obj,k,options)
-                    case _:
-                        setattr(poll_obj,k,v)
```

Comment:
@kallyas I hadn't seen it earlier but your previous solution was alright. We need to split up the list and parse the internals the rebuild the new parsed content. But here the parse method shouldn't be `_parse_polloption`, it should be `_parse_poll` since it's responsible for parsing every object in the `poll.py` module

Reply:
fixed